templates/glue-job-python-shell.yaml

```diff
@@ -83,6 +101,6 @@ Resources:
 Outputs:
 
   JobName:
-    Value: !Ref GlueJob
+    Value: !If [CreateResource, !Ref GlueJob, "GlueJobPythonShellNotCreated"]
```

Comment:
Use `!Ref 'AWS::NoValue'` in the case we do not create the resource

Reply:
Yeah I tried that originally. Apparently in Outputs, they don't allow a `!Ref 'AWS::NoValue'`, it has to be a string. 

Here was the full message: `"An error occurred (ValidationError) when calling the UpdateStack operation: Template format error: The Value field of every Outputs member must evaluate to a String."`